packages/url_launcher/url_launcher_web/lib/src/link.dart

```diff
@@ -192,6 +208,27 @@ class LinkViewController extends PlatformViewController {
     await SystemChannels.platform_views.invokeMethod<void>('create', args);
   }
 
+  void _onDomKeydown() {
```

Comment:
Does this method need to check if the keydown is "enter", "space" or something like that? This'd trigger the link pressing any key, right?

Reply:
I don't want to assume what keys the framework and/or app are using as a trigger for the link.

As I explained in the PR description, there are 2 conditions that have to be met in order for the link to navigate. The code in `_onDomKeydown` is only one part. The other part is the app/framework calling `followLink` when they decide that the key should trigger the link. Here's roughly the sequence of events:

1. User tabs through the app until the focus reaches the button wrapped by the Link widget.
    * (Keep in mind that this tabbing is happening purely in the framework; the browser has no idea what's focused)
2. User hits "Enter".
3. The framework received the keydown event.
    1. It decides to trigger the `onPressed` on the button (because the button has focus).
    2. The button's `onPressed` callback leads to a call to the Link's `followLink`.
    3. `followLink` calls `LinkViewController.registerHitTest` which sets `_hitTestedViewId` to the Link's platform view ID.
4. The `LinkViewController` also receives keydown event.
    1. It checks if `_hitTestedViewId` is already set (indicating that the framework already decided to trigger the link).
    2. If `_hitTestedViewId` is set, it navigates to that Link's URI.

Reply:
(This should be part of the documentation of the widget :P)

Reply:
Added as comments to the `_onGlobalKeydown` method since that's where this information is relevant.